api/api.py

```diff
@@ -499,3 +501,30 @@ def bids(user, key):
             "Your bid will be confirmed once you scan the QR code.",
         ] + (["Started following the auction."] if started_following else []),
     })
+
+
+@api_blueprint.route('/api/stores/<string:twitter_username>/auctions', methods=['GET'])
```

Comment:
I would just call this `/api/users/<string:twitter_username>/auctions` - since it returns the auctions of a user.